packages/ses/package.json

```
@@ -55,24 +55,35 @@
         "default": "./dist/ses.cjs"
       }
     },
+    "./lockdown-hermes": {
```

1. This would be `require("ses/hermes")` which is fair:

   ```suggestion
       "./hermes": {
   ```

2. This would be `require("ses/ses-hermes")` which at least matches the name of the actual bundle:

   ```suggestion
       "./ses/ses-hermes": {
   ```